build-system/tasks/check-types.js

```diff
@@ -8,13 +8,29 @@ const {compileJison} = require('./compile-jison');
 const {cyan, green} = require('kleur/colors');
 const {execOrThrow} = require('../common/exec');
 const {log} = require('../common/logging');
+const {updateSubpackages} = require('../common/update-packages');
+const fastGlob = require('fast-glob');
```

Comment:
don't we use globby in most places?

Reply:
we moved away from it AFAIK b/c it doesn't support windows